<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1261</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>430</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(255, 0, 153);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;
}</string>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>1301</width>
        <height>721</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/back.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_email">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>310</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid rgb(255, 0, 153); /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>491</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QString imagePath1 = &quot;file:///C:/Users/the cast/Downloads/cnx.png&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/cnx.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>301</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QString imagePath2 = &quot;file:///C:/Users/the cast/Downloads/logo.png&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid rgb(255, 0, 153); /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(230, 23, 147);
font: 15pt &quot;Titillium Web&quot;;
border-radius: 10px; /* Coins arrondis */</string>
      </property>
      <property name="text">
       <string>       Welcome</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>350</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Yu Gothic&quot;;
</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>280</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Yu Gothic&quot;;
</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="forgot_password">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(255, 0, 153);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;
}</string>
      </property>
      <property name="text">
       <string>forgot password ? </string>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>label_5</zorder>
     <zorder>label_2</zorder>
     <zorder>lineEdit_password</zorder>
     <zorder>label_8</zorder>
     <zorder>label_11</zorder>
     <zorder>label_7</zorder>
     <zorder>lineEdit_email</zorder>
     <zorder>pushButton_login</zorder>
     <zorder>forgot_password</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>430</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Yu Gothic&quot;;
</string>
      </property>
      <property name="text">
       <string>Answer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>301</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QString imagePath2 = &quot;file:///C:/Users/the cast/Downloads/logo.png&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>1301</width>
        <height>721</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/back.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_confirm">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>510</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(255, 0, 153);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;
}</string>
      </property>
      <property name="text">
       <string>CONFIRM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_answer">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>460</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid rgb(255, 0, 153); /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>350</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Yu Gothic&quot;;
</string>
      </property>
      <property name="text">
       <string>Question:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>491</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QString imagePath1 = &quot;file:///C:/Users/the cast/Downloads/cnx.png&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/cnx.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(230, 23, 147);
font: 15pt &quot;Titillium Web&quot;;
border-radius: 10px; /* Coins arrondis */</string>
      </property>
      <property name="text">
       <string>Forgot Password</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_question">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>380</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 202, 213);
    border: 2px solid #EF8EBA; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;</string>
      </property>
      <item>
       <property name="text">
        <string>Quelle est votre date de naissance ?</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quel est le titre de votre certification professionnelle la plus récente ?</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quel est le nom de votre superviseur professionnel dans notre entreprise ?</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quelle est la date à laquelle vous avez rejoint notre entreprise ?</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quelle est la spécialisation de votre premier diplôme ?</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>280</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Yu Gothic&quot;;
</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_email_forgot">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>310</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid rgb(255, 0, 153); /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>label_6</zorder>
     <zorder>label_12</zorder>
     <zorder>label_3</zorder>
     <zorder>pushButton_confirm</zorder>
     <zorder>lineEdit_answer</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>comboBox_question</zorder>
     <zorder>label_19</zorder>
     <zorder>lineEdit_email_forgot</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>1301</width>
        <height>721</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/back.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>301</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QString imagePath2 = &quot;file:///C:/Users/the cast/Downloads/logo.png&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>491</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QString imagePath1 = &quot;file:///C:/Users/the cast/Downloads/cnx.png&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/cnx.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(230, 23, 147);
font: 15pt &quot;Titillium Web&quot;;
border-radius: 10px; /* Coins arrondis */</string>
      </property>
      <property name="text">
       <string>Forgot Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_repeat_password">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid rgb(255, 0, 153); /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>350</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Yu Gothic&quot;;
</string>
      </property>
      <property name="text">
       <string>Repeat password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>280</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Yu Gothic&quot;;
</string>
      </property>
      <property name="text">
       <string>Enter New Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_change_password">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>430</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(255, 0, 153);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;
}</string>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_new_password">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>310</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid rgb(255, 0, 153); /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1268</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
